docs/advanced_examples/KNearestNeighbors.ipynb

```diff
@@ -192,7 +192,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "FHE inference execution time: 6.60s per sample\n"
+      "FHE inference execution time: 19.48s per sample\n"
```

Comment:
this is not good, it comes from the fact that we went from 7 to 8 bits in the circuit

Reply:
Yes we should control the max bit-width better but isn't this coming from overflow protection in rounding?